dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -135,6 +145,16 @@ private DialogPlus(Builder builder) {
         this.inAnimationResource = (inAnimation == INVALID) ? getAnimationResource(this.gravity, true) : inAnimation;
         this.outAnimationResource = (outAnimation == INVALID) ? getAnimationResource(this.gravity, false) : outAnimation;
 
+        int minimumMargin = activity.getResources().getDimensionPixelSize(R.dimen.default_center_margin);
+        int marginLeft = builder.marginLeft;
+        int marginTop = builder.marginTop;
+        int marginRight = builder.marginRight;
+        int marginBottom = builder.marginBottom;
+        this.marginLeft = getMargin(this.gravity, marginLeft, minimumMargin);
```

Comment:
- I would remove marginleft, top, right and bottom local variables. We can use directly builder.marginRight since we access it directly.  
- I would also remove all "this" from the variables. It seems they are not necessary. What do you think? The code would be more simple

<pre>
marginLeft = getMargin(gravity, builder.marginLeft, minimumMargin);
</pre>